python/py-reflink/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-reflink
+version             0.2.1
+revision            0
+
+categories-append   devel
+platforms           darwin
+license             MIT
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         Python reflink wraps around platform specific reflink implementations
+long_description    ${description}
+
+homepage            https://gitlab.com/rubdos/pyreflink
+
+checksums           rmd160  59ff15dd3f67016c3b509f9c24517dda60595c5b \
+                    sha256  c9253582db24413bfd703abfc1b2a49de78f31b4907239f286e9a1929a1e6e3a \
+                    size    14638
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-pip \
+                    port:py${python.version}-cffi \
```

Comment:
I think `py-cffi` is a `lib` dependency, and not only needed at `build` time.

Reply:
good catch!